compiler/passes/normalize.cpp

```diff
@@ -1751,14 +1751,15 @@ static void change_method_into_constructor(FnSymbol* fn) {
 
   // Now check that the function name matches the name of the type
   // attached to 'this'.
```

Comment:
It's arguably a "style thing" but I try to avoid the expressions if (strcmp(...)) and if (!strcmp(..))   It leads one to think
of strcmp as a function that returns a bool where true means the strings are the same.  In this example I had to
pause for a couple of seconds to convince myself that the logic was the right way around.  I do not believe I would
have hesitated if it had said

if (strcmp() != 0 && strcmp() != 0) return;